datacenter/dtr/2.2/guides/admin/upgrade/index.md

```diff
@@ -4,34 +4,61 @@ keywords: docker, dtr, upgrade, install
 title: Upgrade DTR
 ---
 
-The first step in upgrading to a new minor version or patch release of DTR 2.2,
-is ensuring you're running DTR 2.1. If that's not the case, start by upgrading
-your installation to version 2.1, and then upgrade to 2.2.
+DTR uses [semantic versioning](http://semver.org/) and we aim to achieve specific
```

Comment:
The left nav now has a section just about upgrading, so it makes sense for us to spin this information about upgrade policies into it's own article.

![1](https://cloud.githubusercontent.com/assets/1525937/23087522/ca9dc41c-f529-11e6-9483-b5b847fee192.png)